test/fixtures/e2e/amp-carousel/0.1/slidescroll-autoplay.html

```diff
@@ -44,7 +44,8 @@ <h1>Autoplay slides</h1>
       height="225"
       layout="fixed"
       type="slides"
-      autoplay="3000"
+      autoplay
+      delay="1"
```

Comment:
Note that when `delay` is excluded its default value is `5000` -- so while the `3000` value should be removed from `autoplay` for being ineffective, the test was likely taking the longest because it was waiting for 5s.

Another point - Can the `delay` keep the `3000` value, or go with `1000` or `2000` instead? There are other tests in the same test suite that rely on autoplay not to occur immediately, so `1` is likely too narrow of a window.

Reply:
Yeah the default value of `5000` was definitely what made this stand out, but `3000` is still a long time to wait given that the average test runs in ~`1000`ms. Lets try `1000` for now, but if that seems stable I would like to experiment with decreasing it further in the future.

Reply:
Sounds good, thanks.